sdk/release/src/Upload.hs

```diff
@@ -66,8 +66,9 @@ uploadToMavenCentral MavenUploadConfig{..} releaseDir artifacts = do
     manager <- liftIO $ newManager managerSettings { managerResponseTimeout = responseTimeoutMicro (5 * 60 * 1000 * 1000) }
 
     parsedUrlRequest <- parseUrlThrow $ T.unpack mucUrl -- Note: Will throw exception on non-2XX responses
-    let bearerToken = Base64.encode $ encodeUtf8 (mucUser <> ":" <> mucPassword)
-        baseRequest = setRequestMethod "PUT" $ setRequestBearerAuth bearerToken parsedUrlRequest
+    let baseRequest = setRequestMethod "PUT" 
+            $ setRequestBasicAuth (encodeUtf8 mucUser) (encodeUtf8 mucPassword) 
```

Comment:
not a blocker - but probably good security practice to switch this to bearer token at next available opportunity too